sagemaker-training-compiler/huggingface/pytorch_tune_batch_size/batch_tune.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        entry_point="find_max_batch_size.py",  # Wrapper around training script that enables multi GPU training
</pre><p>Change the comment to something like:</p><pre class="ql-syntax" spellcheck="false"># Wrapper around training script that finds the maximum batch size (and enables multi GPU training?)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>